rele/client.py

```diff
@@ -95,11 +97,33 @@ def _create_topic(self, topic_path):
             request={
                 "name": topic_path,
                 "message_storage_policy": MessageStoragePolicy(
-                    {"allowed_persistence_regions": [self._message_storage_policy]}
+                    {"allowed_persistence_regions": self._message_storage_policy}
                 ),
             }
         )
 
+    def _normalize_storage_policy(self, policy):
+        """
+        Normalizes the storage policy to accept either a string or a list.
+        If it is a string, it converts it into a list with a single element.
+        """
+
+        if isinstance(policy, str):
+            # Warning log for future compatibility
+            print(
+                "Warning: `message_storage_policy` as a string is deprecated. Use a list of regions instead."
+            )
```

Comment:
```suggestion
            warnings.warn(
                "`message_storage_policy` as a string is deprecated. Use a list of regions instead.",
                DeprecationWarning,
            )
```
https://docs.python.org/3.12/library/warnings.html